tests/data/dummy_connection/connection.yaml

```diff
@@ -27,3 +27,14 @@ dependencies:
   dep4:
     version: <=1.11.12b2
 is_abstract: false
+cert_requests:
+- identifier: cert_id
+  not_after: '2020-01-02T00:00:00'
+  not_before: '2020-01-01T00:00:00'
+  path: some/path_1
+  public_key: key_id
```

Comment:
What's `key_id` here?

Reply:
the crypto identifier

Reply:
but we also have `ledger_id`....

Reply:
`public_key` is:
- either a public key
- or a key identifier to get the connection key from `connection_key_paths`. 

- Does that make sense?